### PR TITLE
Test and future parser updates

### PR DESCRIPTION
**Pull request by cmurphy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,28 @@
 ---
+sudo: false
```
Comment (igalic) [inline .travis.yml]: \o/ finally

File: manifests/dev.pp [hunk 2]
```diff
@@ -4,8 +4,10 @@
   }
   include ::apache::params
   $packages = $::apache::params::dev_packages
-  package { $packages:
-    ensure  => present,
-    require => Package['httpd'],
+  if $packages { # FreeBSD doesn't have dev packages to install
+    package { $packages:
```
Comment (igalic) [inline manifests/dev.pp]: i used this as horrible, horrible hack in one of my modules…

File: spec/classes/apache_spec.rb [hunk 3]
```diff
@@ -794,7 +794,7 @@
 
     it do
       expect {
-       should compile
+       catalogue
```
Comment (igalic) [inline spec/classes/apache_spec.rb]: where's that from?

Comment (mhaskel) [inline spec/classes/apache_spec.rb]: undocumented in rspec-puppet? @hunner found it.

Comment (igalic) [inline spec/classes/apache_spec.rb]: how is the undocumented thing better?

**Pull request merged**